cfgov/unprocessed/js/organisms/MegaMenuMobile.js

```diff
@@ -74,17 +74,16 @@ function MegaMenuMobile( menus ) {
    * @param {Event} event - A FlyoutMenu event.
    */
   function handleEvent( event ) {
-    if ( !_suspended ) {
-      if ( event.type === 'triggerClick' ) {
-        _handleTriggerClickBinded( event );
-      } else if ( event.type === 'expandBegin' ) {
-        _handleExpandBeginBinded( event );
-      } else if ( event.type === 'collapseBegin' ) {
-        _handleCollapseBeginBinded( event );
-      } else if ( event.type === 'collapseEnd' ) {
-        _handleCollapseEndBinded( event );
-      }
-    }
+    if ( _suspended ) { return; }
```

Comment:
Fixed! We are allowing single line conditionals to not have braces [here](https://github.com/cfpb/cfgov-refresh/blob/flapjack/.eslintrc#L540). We could swap that to false, but will need to add braces in a few places.